lib/commands/smapi/smapi-command-handler.js

```diff
@@ -38,7 +38,7 @@ const _mapToParams = (optionsValues, flatParamsMap, commanderToApiCustomizationM
         const param = flatParamsMap.get(apiName);
         if (param) {
             let value = optionsValues[key];
-            value = param.isArray ? value.split(ARRAY_SPLIT_DELIMITER) : value;
+            value = (param.isArray && !param.json) ? value.split(ARRAY_SPLIT_DELIMITER) : value;
```

Comment:
I think it is better just change in 1 place
https://github.com/alexa/ask-cli/blob/develop/lib/commands/smapi/cli-customization-processor.js#L85
const isArray = customizedParameter.type === 'array' && !customizedParameter.json